src/drivers/common/periph_memory_alloc.h

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file
+ *
+ * @date Mar 12, 2020
+ * @author Anton Bondarev
+ */
+
+#ifndef SRC_DRIVERS_COMMON_PERIPH_MEMORY_ALLOC_H_
+#define SRC_DRIVERS_COMMON_PERIPH_MEMORY_ALLOC_H_
+
+extern void *periph_memory_alloc(size_t len);
```

Comment:
Include stddef.h to make sure size_t is defined, please

Reply:
Thanks, added